make/istio/run

```diff
@@ -16,24 +16,68 @@ if [ ! -d "$istio_path" ]; then
     | tar zx -C "${istio_path}" --strip-components=1
 fi
 
+istio_init_chart="${istio_path}/install/kubernetes/helm/istio-init"
+istio_chart="${istio_path}/install/kubernetes/helm/istio"
 
-chart="${istio_path}/install/kubernetes/helm/istio"
+# Install all the Istio Custom Resource Definitions.
+istio_init_args=(
+  "${ISTIO_INIT_HELM_RELEASE}"
+  --namespace "${ISTIO_NAMESPACE}"
+  --timeout 900 
+)
+helm upgrade --install "${istio_init_args[@]}" "${istio_init_chart}"
+
+max_attempts=5
+for (( i=1; i<=max_attempts; i++ ))
+do
+  echo "[INFO] Check if istio required CRDs are created"
+  set +e
+  crd_num=$(kubectl get crds | grep -c 'istio.io\|certmanager.k8s.io')
+  set -e
+  if [ "${crd_num}" == "53" ]; then
+    echo "[INFO] 53 CRDs are installed successfully"
+    break
+  elif [ ${i} -eq ${max_attempts} ]; then
+    echo "[ERROR] Failed to install CRDs after retry ${max_attempts} times" >&2
+    exit 1
+  else
+    echo "[INFO] Waiting for creating CRDs, sleep another 2s ..." >&2
+    sleep 2
+  fi
+done
 
 # Install the Istio Helm chart.
-args=(
+# Create kiali secret
+KIALI_USERNAME=$(echo -n "admin" | base64)
```

Comment:
I think `KIALI_USERNAME` and `KIALI_PASSPHRASE` should be environment variables that we allow to be passed to the script.

This can be done with:

```shell
kiali_username_encoded=$(echo -n "${KIALI_USERNAME:-admin}" | base64)
kiali_passphrase_encoded=$(echo -n "${KIALI_PASSPHRASE:-admin}" | base64)
```